client/src/components/Login/LoginForm.vue

```diff
@@ -49,18 +49,21 @@ const props = withDefaults(defineProps<Props>(), {
 
 const router = useRouter();
 
-const urlParams = new URLSearchParams(window.location.search);
+const getUrlParams = () => new URLSearchParams(window.location.search);
+const params = getUrlParams();
```

Comment:
Was there a reason to decompose the original line this way?

Reply:
removed